jenkins/job-configs/kubernetes-jenkins-pull/kubernetes-pull.yaml

```diff
@@ -51,6 +52,7 @@
         - github-pull-request:
             # This is the Jenkins GHPRB plugin ID, not the actual github token.
             auth-id: 'f8e31bc1-9abb-460a-a2ca-9c4aae3ca4e8'
+            only-trigger-phrase: '{only-trigger-phrase}'
```

Comment:
@spxtr This does not seem to be true. I tested it out in https://github.com/kubernetes/test-infra/pull/359 by setting only-trigger-phrase to false explicitly for build-test-e2e-gke project. The value of onlyTriggerPhrase remained false in the generated XML.